### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by roscha80**
Added Card but it wont work yet. Don't know see the failure

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-      <Button isActive>Text dazwischen</Button>
-      <Button>Text dazwischen</Button>
-    </div>
+    <section>
+      <div>
+        <Header isBackButtonVisible>Header with Button</Header>
+        <Header>Another Header</Header>
+      </div>
+      <div>
+        <Card>Card</Card>
```
Comment (czintel) [inline src/App.jsx:15]: Im Card-Tag musst du noch deine in der Cards.jsx definierten Parameter und deren Attribute übergeben. Das Tag muss zudem self-closing sein.

Und dann das ganze nochmal mit "isBookmarked"

File: src/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import './Card.css'
+
+export default function Card(title, text, isBookmarked) {
```
Comment (macfrei) [inline src/Card.jsx:4]: We use destructuring assignment to get the properties `title`, `text` and `isBookmarked` from the `props` object we get as a parameter from React. It's either:
```js
export default function Card(props) {
    const {title, text, isBookmarked} = props
```
Or:
```suggestion
export default function Card({title, text, isBookmarked}) {
```
And that is the same as if we'd write it like this: `const {title, text, isBookmarked} = props`

File: src/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <section className="Card">
+      <div className={isBookmarked ? 'Card bookmark' : 'Card'} />
```
Comment (macfrei) [inline src/Card.jsx:7]: You need to add the `bookmark_added` class if it is added, like an active class?

**Review by macfrei — COMMENTED** [review]
Good job, added a few comments :)

**Pull request merged**